airflow/models/__init__.py

```diff
@@ -69,6 +69,7 @@ def import_all_models():
     import airflow.models.dataset
     import airflow.models.serialized_dag
     import airflow.models.tasklog
+    import airflow.www.fab_security.sqla.models
```

Comment:
I'm wondering if we should do this after 2.5.0 release.

There are also some other things to consider e.g:
https://github.com/apache/airflow/blob/fb7c6afc8cb7f93909bd2e654ea185eb6abcc1ea/airflow/utils/db.py#L680
and 
https://github.com/apache/airflow/blob/fb7c6afc8cb7f93909bd2e654ea185eb6abcc1ea/airflow/www/fab_security/sqla/models.py#L50

Reply:
Yes, I think we can further simplify, and I would prefer to wait until after as well, but the notes PR needs this change now.

Do you think I should try and do it all now, or just the minimal simplification?

Reply:
We can go on with it